ringvax/app.py

```diff
@@ -362,44 +376,30 @@ def infectiousness_callback():
                 help=f"The probability that there are no infections in the {format_control_gens(control_generations)}, or equivalently that the {format_control_gens(control_generations - 1)} do not produce any further infections.",
             )
 
-            st.header("Number of infections")
-            with st.expander("Plot Options"):
-                generational_counts = get_generational_infection_count_df(sim_df)
-                plot_gen = st.segmented_control(
-                    "Generation to plot",
-                    options=range(1, n_generations + 1),
-                    default=n_generations,
-                )
-                cumulative = (
-                    True
-                    if st.segmented_control(
-                        "Show infections cumulatively or in specific generation?",
-                        options=["Cumulative", "In generation"],
-                        default="Cumulative",
-                    )
-                    == "Cumulative"
-                    else False
+            st.header(
+                "Number of infections",
+                help="This is a histogram describing the distribution of the number of infections. You can change what is plotted here in the Advanced Settings.",
+            )
+            generational_counts = get_infection_counts_by_generation(sim_df)
+
+            if cumulative:
+                counts = (
+                    generational_counts.filter(pl.col("generation") <= plot_gen)
+                    .group_by("simulation")
+                    .agg(pl.col("num_infections").sum())
                 )
+            else:
+                counts = generational_counts.filter(pl.col("generation") == plot_gen)
 
-                if cumulative:
-                    counts = (
-                        generational_counts.filter(pl.col("generation") <= plot_gen)
-                        .group_by("simulation")
-                        .agg(pl.col("count").sum())
-                    )
-                else:
-                    counts = generational_counts.filter(
-                        pl.col("generation") == plot_gen
-                    )
-
-            size_hist_title = f"Distribution of the {'cumulative ' if cumulative else ''}number of infections seen {'up to and including' if cumulative else 'in'} generation {plot_gen}."
-            st.write(size_hist_title)
+            x_lab = f"Number of infections in generation {plot_gen}"
+            if cumulative:
+                x_lab = f"Cumulative infections through generation {plot_gen}"
```

Comment:
```suggestion
            x_lab = (
                f"Cumulative infections through generation {plot_gen}"
                if cumulative
                else f"Number of infections in generation {plot_gen}"
            )
```
just a suggestion; I find it easier to read this as "`x_lab` must be or the other"